includes/AMP/Meta_Sanitizer.php

```diff
@@ -116,7 +116,7 @@ protected function ensure_boilerplate_is_present() {
 	 *
 	 * @return string[] Stylesheets, where first is contained in style[amp-boilerplate] and the second in noscript>style[amp-boilerplate].
```

Comment:
doc block says string here - vs array

Reply:
This is unrelated to my change and should be done in another PR

Reply:
This means an array of strings `string[]` doesn't it?

Reply:
Your right I misread that